src/merge-link.vue

```diff
@@ -22,6 +22,28 @@ export default {
     onSuccess: Function,
     onExit: Function,
     tenantConfig: Object,
+    onValidationError: Function,
+    filePickerConfig: {
+      type: Object,
+      required: false,
+      validator: function (filePickerConfigValue) {
```

Comment:
Does the function need to be defined here? This looks like it's just a type config - could it be : Function like the others, or is the implementation needed?

Reply:
ah this isnt the function definition its just a validator to make sure the object is typed right (filePickerConfig) 

yeah tbh not sure what best practices are i could have left it just as filePickerConfig: Object but that felt so loose LOL